cli/uninstall/uninstall.go

```diff
@@ -79,6 +80,27 @@ func UninstallProgram(program string, programVersion string, binDst string, head
 	log.Infof("Removing binary...")
 	var err error
 
+	if program == search.ProgramDev {
+		tarantoolBinary := filepath.Join(binDst, "tarantool")
+		binActive, err := util.ResolveSymlink(tarantoolBinary)
+		if err != nil {
+			return err
+		}
+		if !version.IsTarantoolDev(binActive, binDst) {
+			return fmt.Errorf("%s is no installed", program)
+		}
+		if err := os.Remove(tarantoolBinary); err != nil {
```

Comment:
Can we name it tarantoolBinary -> tarantoolBinarySymlink, because `os.Remove(tarantoolBinary)` looks like we are removing binary, which is not true.